module/data/chat-message/actorRoll.mjs

```diff
@@ -17,6 +17,16 @@ const targetsField = () =>
         })
     );
 
+export const itemSourceField = () =>
+    new fields.SchemaField({
+        type: new fields.StringField({
+            choices: CONFIG.DH.ITEM.itemSourceType,
+            initial: CONFIG.DH.ITEM.itemSourceType.itemCollection
+        }),
+        itemPath: new fields.StringField(),
+        actionIndex: new fields.StringField()
+    });
+
```

Comment:
`itemPath` points the way to the downtime item in the game setting in this case, but it's intended to be generic if we expand `type` to more places where actions reside.
`actionIndex` is used, because the actual `id` of the downtime moves are changed on every reload.